### PR TITLE
✨ Add Hammer of Sol

### PR DESCRIPTION
**Pull request by shrinerh**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "onBuildSuccess": null
   },
   "mc": {
-    "dev": true,
+    "dev": false,
```
Comment (satyrnidae) [inline config.json]: Let's keep dev true in the repo cuz I'll probably be debugging something later and wondering why nothing is showing lol

**Review by satyrnidae — CHANGES_REQUESTED** [review]
Looks great, can you change the dev flag back to true for the repo tho?

File: config.json (hunk 1 above)
Comment (shrinerh) [reply config.json]: done

**Review by satyrnidae — APPROVED** [review]
ty ty

**Pull request merged**